examples/speaker_recognition/spkr_get_emb.py

```diff
@@ -0,0 +1,66 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+import pytorch_lightning as pl
+from pytorch_lightning import seed_everything
+
+from nemo.collections.asr.models import ExtractSpeakerEmbeddingsModel
+from nemo.core.config import hydra_runner
+from nemo.utils import logging
+from nemo.utils.exp_manager import exp_manager
+
+os.environ["TOKENIZERS_PARALLELISM"] = "false"
```

Comment:
Dont go setting this in code, it overrides user override from command line. Pass it explicitly in the execution example.

Reply:
At the very lease check here if the user has already set this value, and default to false if they have not.